lib/src/icons/validationcheck.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import './validationcheck.scss';
+
+const ValidationCheck = () =>
+  <svg viewBox="0 0 16 16" className="nx-icon nx-icon--validation-check">
+    <path fill="currentColor"
+          d="M13.7142784,4 L16,6.25 L6.09520643,16 L3.80948484,13.75 L3.81,13.749 L-5.32907052e-14,9.99999135
+             L2.28572159,7.74999135 L6.095,11.5 L13.7142784,4 Z">
```

Comment:
The FA check wasn't good?

Reply:
> The FA check wasn't good?

FA check is good to use, when I adjust the dimensions of the icons however, I consider them custom at that point. I guess the question is, does it matter if I tweak the height/width. Reason being is that some of the icons appear smaller/larger than others and I would like to try and keep them all as uniform as possible

Reply:
> I guess the question is, does it matter if I tweak the height/width.

We use SVG for all icons, so scaling is not considered a separate icon.  Even if you are talking about changing the aspect ratio, If it's still ultimately based on the FA icon I'd rather use said FA icon along with a `transform` rather than created a completely custom SVG that essentially duplicates the FA one.